ui/src/app/shared/utils/url-search-params.utils.ts

```diff
@@ -154,7 +154,14 @@ import {TimeFrame} from "../model/TimeFrame";
     let queryFields = new Map<string, FieldDataType>(queryDataTypes);
 
     if (capabilities.commonLabels) {
-      capabilities.commonLabels.map(label => queryFields.set(label, FieldDataType.Text));
+      capabilities.commonLabels.map(label => {
+        let currentField = capabilities.displayFields.find( f => f.field == 'labels.' + label);
+        if (currentField && currentField.validFieldValues) {
+          queryFields.set(label, FieldDataType.Enum);
```

Comment:
The function doc should be updated, currently says "non-label fields"

Reply:
nice catch – updated the comment